rocAL/rocAL_pybind/example/new_api/rocAL_api_python_unittest.py

```diff
@@ -16,8 +16,11 @@ def draw_patches(img, idx, device):
             image = img.detach().numpy()
     else:
         image = img.cpu().numpy()
-    image = image.transpose([1, 2, 0])
     image = cv2.cvtColor(image, cv2.COLOR_RGB2BGR)
+    if not args.NHWC:
+        image = image.transpose([1, 2, 0])
+    if args.fp16:
+        image = (image).astype('uint8')
```

Comment:
Is this correct. For fp16, we are converting to uint8

Reply:
I think @rrawther  that's for display, I was going to comment too

Reply:
Its for opencv imwrite.

we need to convert the fp16 to uint8 for dumping images.